scripts/linux/runE2ETest.sh

```diff
@@ -1074,7 +1074,7 @@ function usage() {
     echo ' -eventHubConsumerGroupId                       Optional Event Hub Consumer Group ID for the Analyzer module.'
     echo ' -loadGenMessageFrequency                       Frequency to send messages in LoadGen module for long haul and stress test. Default is 00.00.01 for long haul and 00:00:00.03 for stress test.'
     echo ' -snitchAlertUrl                                Alert Url pointing to Azure Logic App for email preparation and sending for long haul and stress test.'
-    echo ' -snitchBuildNumber                             Build number for snitcher docker image for long haul and stress test. Default is 1.1.'
+    echo ' -snitchBuildNumber                             Build number for snitcher docker image for long haul and stress test. Default is 1.4.'
```

Comment:
I didn't see this var in any of the pipeline variables so no need to trawl through all of them and update :)